src/components/KInlineEdit/KInlineEdit.vue

```diff
@@ -22,102 +22,96 @@
+const inputText = ref('')
+const styles = ref({})
+
+const handleClick = async (e: any): Promise<void> => {
+  // If clicking the slot wrapper lets exit out as to not
+  // copy its styles to the input
+  if (e.target.id === 'element-content-wrapper') return
+
+  // Get current STYLES off of the element
+  // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+  // @ts-ignore
+  styles.value = { ...getStyles(e.target), ...props.styleOverrides }
+  inputText.value = props.ignoreValue ? '' : e.target.textContent
+  isEditing.value = true
+
+  // Wait for Vue to update styles & text
+  await nextTick()
+
+  if (input.value) {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
+    // @ts-ignore
+    input.value.focus()
+  }
 }
 
-export default defineComponent({
-  name: 'KInlineEdit',
-  props: {
-    /**
-     * Sets whether the initial value passed in should be ignored.
-     * Useful for times when you have placeholder text and don't want it passed
-     * in as a value
-     */
-    ignoreValue: {
-      type: Boolean,
-      default: false,
-    },
-    /**
-     * An object of css styles to override those plucked from the slot element.
-     * Useful if you are styling the default content differently than how the
-     * input should display
-     */
-    styleOverrides: {
-      type: Object,
-      default: () => ({}),
-    },
-  },
-  emits: ['changed'],
-  setup(props, { emit, slots }) {
-    const input = ref(null)
-    const inputUuid = computed((): string => 'editable-wrapper-' + uuidv1())
-    const isEditing = ref(false)
-    const inputText = ref('')
-    const styles = ref({})
-
-    const handleClick = async (e: any): Promise<void> => {
-      // If clicking the slot wrapper lets exit out as to not
-      // copy its styles to the input
-      if (e.target.id === 'element-content-wrapper') return
-
-      // Get current STYLES off of the element
-      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-      // @ts-ignore
-      styles.value = { ...getStyles(e.target), ...props.styleOverrides }
-      inputText.value = props.ignoreValue ? '' : e.target.textContent
-      isEditing.value = true
-
-      // Wait for Vue to update styles & text
-      await nextTick()
-
-      if (input.value) {
-        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
-        // @ts-ignore
-        input.value.focus()
-      }
-    }
+const handleSave = (): void => {
+  isEditing.value = false
+  emit('changed', inputText.value)
+}
 
-    const handleSave = (): void => {
-      isEditing.value = false
-      emit('changed', inputText.value)
-    }
+const getStyles = (element: HTMLElement) => {
+  const elementStyles = getComputedStyle(element)
 
-    const getStyles = (element: HTMLElement) => {
-      const elementStyles = getComputedStyle(element)
+  return Object.keys(Styles).reduce((acc: string, cur: string) => {
+    // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
question: did try to see if we could fix whatever issue we were trying to ignore previously?

Reply:
I think it can be fixed in task with adding types